core/src/test/java/org/elasticsearch/search/suggest/SuggestTests.java

```diff
@@ -90,7 +90,7 @@ public void testFromXContent() throws IOException {
         Suggest suggest = createTestItem();
         XContentType xContentType = randomFrom(XContentType.values());
         boolean humanReadable = randomBoolean();
-        BytesReference originalBytes = toXContent(suggest, xContentType, params, humanReadable);
+        BytesReference originalBytes = toShuffledXContent(suggest, xContentType, params, humanReadable);
```

Comment:
Note: this test doesn't need any field randomization I think. The suggest element is a object containing names Suggestions, which we already test elsewhere. I think it is not possible that we add anything other inside this object unless we change the whole structure of the xContent (which is another story).